docs/algorithm.md

```diff
@@ -1,120 +1,152 @@
-# Deterministic algorithm
+# Deterministic Algorithm
 
-Nowaday we can use Machine Learning to translate sentences from one language to other, using a the `sequence to sequence` model. But in some cases there are a small problem: not enough training data.
+Nowaday we can use Machine Learning to translate sentences from one language to other, using a the `sequence to sequence` model. But in some cases there are a little problem: _not enough training data_.
 
-This was our situation. We tried to translate _Spanish_ sentences to _Kaqchikel_. But we could not find enough data to train a model. So we make a decision: 
+That was what happened to us. We tried to translate _Spanish_ sentences to _Kaqchikel_ (a _Mayan_ language). But we could not find enough data to train a model. So we make a decision: 
 
-1. Use translation based on rule.
+1. Use translation based on rules.
 
-This will allow to us generate training data to then be able to build a model _se2seq_.
+This approach will allow to us generate training data to then be able to build a _se2seq_ model.
 
 In this section we will describe our translation rules.
 
 ## Rule structure
 
-This is an example of a definiton rule structure.
+This is an example of a rule definition.
 
-```js
+```json
 {
-    "pattern": "DET,NOUN,VERB", // Generated based on the rules
+    "pattern": "VERB,ADV,ADJ",
     "details": [
         {
-            "tag": "DET",
-            "type: "",
+            "tag": "VERB",
+            "type": "",
             "properties": {
```

Comment:
what's the purpose to have properties on details?

Reply:
It will be used to filter by `tense` in verbs for example. It should have all the properties gotten from Linguakit.